vello/src/render.rs

```diff
@@ -131,18 +150,29 @@ impl Render {
             recording.upload_uniform("config", bytemuck::bytes_of(&cpu_config.gpu)),
         );
         let info_bin_data_buf = ResourceProxy::new_buf(
-            buffer_sizes.bin_data.size_in_bytes() as u64,
+            buffer_sizes.bump_buffers.bin_data.size_in_bytes() as u64
+                + (layout.bin_data_start as u64) * std::mem::size_of::<u32>() as u64,
```

Comment:
This is a change of semantics in this PR, to make the reallocation require less context; essentially, I have changed the `bump` count for `bin_data` to only include the "dynamic" portion of the amount, rather than the pre-apportioned.
Otherwise, the reallocation logic at the top of `render_to_surface` would need to retain `bin_data_start` from the previous frame to actually know the correct value.
I believe there's a corresponding change elsewhere, although I might have forgotten to add that